source/ProgArgs.cpp

```diff
@@ -488,6 +488,20 @@ void ProgArgs::defineAllowedArgs()
 /*s3b*/	(ARG_S3BUCKETACLPUT_LONG, bpo::bool_switch(&this->runS3BucketAclPut),
 			"Put S3 bucket ACLs. This requires definition of grantee, grantee type and "
 			"permissions.")
+/*s3b*/	(ARG_S3STATDIRS_LONG, bpo::bool_switch(&this->runS3StatDirs),
+			"Do bucket Stats.")
+/*s3b*/	(ARG_S3_BUCKET_TAG_INLINE, bpo::bool_switch(&this->doS3BucketTagInline),
+			"Activate bucket tagging operations during other benchmarking phases: "
+            "Adding of bucket tags in create bucket phase, "
+            "reading of tags in stat bucket phase and deletion of tags in delete bucket phase.")
+/*s3b*/	(ARG_S3_BUCKET_TAG_INLINE_VERIFY, bpo::bool_switch(&this->doS3BucketTagInlineVerify),
+            "Verify the correctness of S3 bucket tagging results (requires \"--" ARG_S3_BUCKET_TAG_INLINE "\")")
+/*   */	(ARG_S3_OBJECT_TAG_INLINE, bpo::bool_switch(&this->doS3ObjectTagInline),
```

Comment:
this is sorted alphabetically by the option name, so the two "--s3otag..." options would need to get moved a few lines further down. same for "--s3statdirs".